.travis.yml

```diff
@@ -20,7 +23,7 @@ before_install:
     - sh -c "if [ $TRAVIS_PHP_VERSION != 'hhvm' ]; then phpenv config-add ./tests/travis/php.ini; fi"
 
 install:
-    - composer --prefer-source --dev install
+    - composer --prefer-source install
```

Comment:
more normal to add the flags after the command name: `composer install --prefer-source`.